data/grib2_table_4_2_10_0.csv

```diff
@@ -45,7 +45,7 @@
 39,"ALRRC","Altimeter Range Relative Correction","-","UC_NONE"
 40,"MNWSOW","10 Metre Neutral Wind Speed Over Waves","m/s","UC_NONE"
 41,"MWDIRW","10 Metre Wind Direction Over Waves","Degree true","UC_NONE"
-42,"WESP","Wave Engery Spectrum","s/((m^2)*rad)","UC_NONE"
+42,"WESP","Wave Energy Spectrum","s/((m^2)*rad)","UC_NONE"

```

Comment:
the same change should be done in frmts/grib/degrib/data/ which is one of the sources of that generated file